### PR TITLE
Add `OWC 2022` news post and update wiki links

### PR DESCRIPTION
**Pull request by ChillierPear**
lets see if I get it right this time

### CODE REVIEW

**Review by Walavouchey — COMMENTED** [review]
doesn't the banner already exist in the wiki article?

File: news/2022-09-15-osu-world-cup-2022-registrations-now-open.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+| Round of 32 | Oct 22 & Oct 23 |
+| Round of 16 | Oct 29 & Oct 30 |
+| Quarterfinals | Nov 05 & Nov 06 |
+| Semifinals | Nov 12 & Nov 13 |
+| Finals | Nov 19 & Nov 20 |
+| Grand Finals | Nov 26 & Nov 27 |
+
+## Prizes
+
+The osu! World Cup 2022 offers a $5,000 prize pool, along with limited edition merch! The prize pool may increase further from **[community donations](https://opencollective.com/osu-tournaments/events/osu-world-cup-2022-0b3a1a25)**!
```
[the remark this answers is not in the document]
Comment (ChillierPear) [reply news/2022-09-15-osu-world-cup-2022-registrations-now-open.md:37]: no joiner for consistency, all other 2022 posts did not include one

[the remark this answers is not in the document]
Comment (Walavouchey) [reply news/2022-09-15-osu-world-cup-2022-registrations-now-open.md:37]: was looking at the paragraph below, but i think you're correct anyway

mind editing that on the article as well?

[the remark this answers is not in the document]
Comment (ChillierPear) [reply news/2022-09-15-osu-world-cup-2022-registrations-now-open.md:37]: will leave this then, wiki update added

File: news/2022-09-15-osu-world-cup-2022-registrations-now-open.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+- **1st:** 48% of the prize pool, exclusive limited-edition merch, profile badge, "osu! Champion" user title for one year
```
[the remark this answers is not in the document]
Comment (ChillierPear) [reply news/2022-09-15-osu-world-cup-2022-registrations-now-open.md]: updated with the conventions from wiki entry

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**